launcher/ui/pages/instance/StoragePage.cpp

```diff
@@ -95,34 +92,34 @@ void StoragePage::retranslate()
     ui->retranslateUi(this);
 }
 
-void StoragePage::HandleClearScreenshotsButton()
+void StoragePage::handleClearScreenshotsButton()
 {
-    auto path = (m_inst->gameRoot() + "/screenshots").toStdString();
-    clear_directory_inner(path);
-    update_calculations();
+    auto path = (m_inst->gameRoot() + "/screenshots");
+    clearDirectoryInner(path);
+    updateCalculations();
 }
 
-void StoragePage::HandleClearLogsButton()
+void StoragePage::handleClearLogsButton()
 {
-    auto path = (m_inst->gameRoot() + "/logs").toStdString();
-    clear_directory_inner(path);
-    update_calculations();
+    auto path = (m_inst->gameRoot() + "/logs");
+    clearDirectoryInner(path);
+    updateCalculations();
 }
 
-void StoragePage::HandleClearAllButton()
+void StoragePage::handleClearAllButton()
 {
     HandleClearScreenshotsButton();
     HandleClearLogsButton();
 }
 
-void StoragePage::update_calculations()
+void StoragePage::updateCalculations()
 {
-    m_size_resource_packs = get_directory_size((m_inst->gameRoot() + "/texturepacks").toStdString()) +
-                            get_directory_size((m_inst->gameRoot() + "/resourcepacks").toStdString());
-    m_size_mods = get_directory_size(m_inst->modsRoot().toStdString());
-    m_size_saves = get_directory_size((m_inst->gameRoot() + "/saves").toStdString());
-    m_size_screenshots = get_directory_size((m_inst->gameRoot() + "/screenshots").toStdString());
-    m_size_logs = get_directory_size((m_inst->gameRoot() + "/logs").toStdString());
+    m_size_resource_packs =
+        getDirectorySize((m_inst->gameRoot() + "/texturepacks")) + getDirectorySize((m_inst->gameRoot() + "/resourcepacks"));
+    m_size_mods = getDirectorySize(m_inst->modsRoot());
+    m_size_saves = getDirectorySize((m_inst->gameRoot() + "/saves"));
+    m_size_screenshots = getDirectorySize((m_inst->gameRoot() + "/screenshots"));
+    m_size_logs = getDirectorySize((m_inst->gameRoot() + "/logs"));
 
     fs::space_info space_info = fs::space(m_inst->gameRoot().toStdString());
     m_size_remaining = space_info.free;
```

Comment:
This could be changed to use QStorageInfo, then we don't need to use std::string anywhere anymore.